<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browseButton1">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>10</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>choose folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseButton3">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>90</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>save location</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>160</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>40</y>
      <width>621</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="vendor_lineedit">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>290</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>اسم المورد</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>330</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>الرقم الضريبي</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vatid_lineedit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>370</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>التاريخ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="date_lineedit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="total_lineedit">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>410</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>مبلغ الفاتورة</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vatamount_lineedit">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>450</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>مبلغ الضريبة</string>
    </property>
   </widget>
   <widget class="QPushButton" name="last">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="first">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>save invoice</string>
    </property>
   </widget>
   <widget class="QLabel" name="folder_path_label">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>50</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="location_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ai_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ai extract</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
